controllers/managedocs_controller.go

```diff
@@ -207,6 +207,8 @@ func (r *ManagedOCSReconciler) SetupWithManager(mgr ctrl.Manager, ctrlOptions *c
 					}
 				} else if name == rookConfigMapName {
 					return true
+				} else if name == utils.DataConfigMapName {
```

Comment:
The name `DataConfigMapName` is very generic and does not convey intent
```suggestion
				} else if name == utils.IMDSConfigMapName {
```